bin/metadata.js

```diff
@@ -56,12 +56,7 @@ async function main(prid, owner, repo) {
   logger.info({ raw: metadata }, 'Generated metadata:');
 
   const checker = new PRChecker(pr, reviewers, comments, reviews);
-  checker.checkReviewers();
-  checker.checkReviews();
-  checker.checkPRWait();
-  checker.checkCI();
-  // TODO: check committers against authors
-  // TODO: maybe invalidate review after new commits?
+  checker.check();
```

Comment:
Maybe `checkAll()`? That would make the tests easier to read a bit (I am working on some tests now)